### PR TITLE
Setup & Mobile for Porfolio Files

### PR DESCRIPTION
**Pull request by okemboi**
In this branch, 
- I created the index html file as the homepage of the portfolio website.
- I added added the html header element which contains header meta tags.
- I added the body element which includes, a second header element, the title, sub title, a paragraph and social media links. 
- I added an external CSS folder which contains a master stylesheet and an image folder to hold all graphics.
- I added a linter that helps to diagnose and fix technical issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <section class="logo"><a href="#">Kemboi</a></section>
+            <section class="menu"><img src="/images/union.svg"></section>
+        </div>
+
+        </div>
+
+    </header>
+    <main>
+        <div>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:25]: - [ ] Please make sure the you remove this empty element to make your code clean. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @okemboi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        <div class="box twitter">
+            <a href="https://twitter.com/ok_kemboi"><img src="/images/twitter.svg" alt="Twitter">
+        </div>
+
+        <div class="box medium">
+            <a href="#"><img src="/images/M.svg" alt="Medium"></a>
+
+        </div>
```
[the remark this answers is not in the document]
Comment (okemboi) [reply index.html]: I've fixed he section tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <div class="box two">
+            <a href="#"><img src="/images/two.svg" alt="Two"></a>
+        </div>
+
+        <div class="box twitter">
+            <a href="https://twitter.com/ok_kemboi"><img src="/images/twitter.svg" alt="Twitter">
+        </div>
+
+        <div class="box medium">
+            <a href="#"><img src="/images/M.svg" alt="Medium"></a>
```
[the remark this answers is not in the document]
Comment (okemboi) [reply index.html]: Ive added the ul and li tags in the social media icons

File: index.html (hunk 1 above)
Comment (okemboi) [reply index.html]: I've removed the empty elements, my code is cleaner. thanks

File: index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <title>Kemboi Portfolio</title>
+</head>
+
+<body>
+    <header>
+
+        <div>
+            <section class="logo"><a href="#">Kemboi</a></section>
+            <section class="menu"><img src="/images/union.svg"></section>
+        </div>
```
[the remark this answers is not in the document]
Comment (okemboi) [reply index.html]: The hamburger button is now in the nav tag

[the remark this answers is not in the document]
Comment (okemboi) [reply index.html]: The hamburger button is now in the nav tag

File: style/style.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+main {
+  background-image: url(/images/illustration.svg);
+  background-repeat: no-repeat;
+  background-size: 156px;
+  background-position-x: 161px;
+  background-position-y: 250px;
+  height: 40rem;
+  position: relative;
+}
```
Comment (Qoosim) [inline style/style.css:53]: - [ ] Great job implementing requested changes by the previous reviewer. However, there's little stuf you still need to work on. At this moment, the background image on the page is not properly position, not responsive. Kindly work on it to make it appear as we have it on the figma design.

### Your Design
[bg.webm](https://user-images.githubusercontent.com/26597432/228916396-ece3cabc-add9-4c51-8090-408af08895f3.webm)

### Figma Design
temp-bg

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @okemboi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style/style.css (hunk 5 above)
Comment (okemboi) [reply style/style.css]: I've positioned the background image properly and made it responsive.

**Review by jerryowusu — APPROVED** [review]
Hi @okemboi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


yeasss


## Highlights
:heavy_check_mark: Nice UI design 
:heavy_check_mark: Great effort in making the requested changes


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — COMMENTED** [review]
Hi @okemboi , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project pre-approved PR for review. Kindly submit a new PR for evry milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by okemboi** [comment]
I’m repeating the Module and I asked the students success who said it was fine to use previous work

Kind regards,
Kemboi Kiptanui

________________________________

**Comment by wayungi** [comment]
Hey @okemboi , the mistake is mine. please let me fixed.

**Comment by okemboi** [comment]
OK. Thank you

Kind regards,
Kemboi Kiptanui

________________________________

**Comment by wayungi** [comment]
Please let me know if it shows approved on your dashboard

**Comment by wayungi** [comment]
Hey @okemboi, is everything okay now?

**Review by souad988 — COMMENTED** [review]
Hi @okemboi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Good efforts
Design well implemented

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by okemboi** [comment]
Thank you

Best Regards
K. Kemboi
________________________________

**Pull request merged**